javaparser-core/src/main/java/com/github/javaparser/JavadocParser.java

```diff
@@ -57,9 +60,15 @@ public static Javadoc parse(String commentContent) {
             blockLines = Collections.emptyList();
         } else {
             descriptionText = trimRight(String.join("\n", cleanLines.subList(0, indexOfFirstBlockTag)));
-            blockLines = cleanLines.subList(indexOfFirstBlockTag, cleanLines.size()).stream()
-                    .filter(Utils.STRING_NOT_EMPTY)
-                    .collect(Collectors.toList());
+            String tagBlock = cleanLines.subList(indexOfFirstBlockTag, cleanLines.size())
```

Comment:
I am a bit confused by the fact you first take everything and put it into a string and later you seem to split it again and remove empty lines. Could you add some comments to explain what is going on.

I mean, I should have written way more comments when I wrote the original version of this cose, but let's not focus on that...